tools/update-prod-static

```diff
@@ -87,12 +87,10 @@ subprocess.check_call(['./manage.py', 'compilemessages'],
 
 # Move the source maps out of the serve/ directory and into their
 # proper place.
-subprocess.check_call(['rm', '-rf', 'prod-static/source-map'],
-                      stdout=fp, stderr=fp)
 subprocess.check_call(['mkdir', '-p', 'prod-static'],  # Needed if PRODUCTION
                       stdout=fp, stderr=fp)
-subprocess.check_call(['mv', os.path.join(settings.STATIC_ROOT, 'source-map'),
-                       'prod-static/source-map'],
+subprocess.check_call(['cp', '-r', os.path.join(settings.STATIC_ROOT, 'source-map'),
+                       'prod-static'],
```

Comment:
Yeah, and I'd take it slightly further, to `cp -aT` -- I think that's the clearest general-purpose "copy everything *here* to be *there*" version of a `cp` command line.

This one will consistently do the right thing anyway because of that `mkdir -p`, but the `-T` makes it more explicit and makes it simpler for the reader to quickly see for sure exactly what's meant to happen, so I think it's a good practice.